### PR TITLE
Support recipes

### PR DESCRIPTION
**Pull request by martonsereg**
@keyki just the usual stuff

### CODE REVIEW

File: src/main/groovy/com/sequenceiq/cloudbreak/client/CloudbreakClient.groovy [hunk 1]
```diff
@@ -130,6 +133,19 @@ class CloudbreakClient {
         return response?.data?.id
     }
 
+    def String postRecipe(String recipe, Boolean publicInAccount) throws Exception {
```
Comment (keyki) [inline src/main/groovy/com/sequenceiq/cloudbreak/client/CloudbreakClient.groovy]: Is this method intended to create a recipe? In that case it is missing lots of fields from the JSON. The only case it is valid if you post the JSON as String in the recipe variable. Is that the case?

Comment (martonsereg) [inline src/main/groovy/com/sequenceiq/cloudbreak/client/CloudbreakClient.groovy]: It is exactly the case, recipes are read from files just like blueprints in the shell. I'll update the client in a later PR to include a more sophisticated method to create recipes.

File: src/main/groovy/com/sequenceiq/cloudbreak/client/CloudbreakClient.groovy [hunk 2]
```diff
@@ -130,6 +133,19 @@ class CloudbreakClient {
...
+        log.debug("Posting recipe to account...")
+        def binding = ["RECIPE": recipe]
+        def response;
+        if (publicInAccount) {
+            response = processPost(Resource.ACCOUNT_RECIPES, binding)
+        } else {
+            response = processPost(Resource.USER_RECIPES, binding)
+        }
+        log.debug("Got response: {}", response.data.id)
+        return response?.data?.id
```
Comment (keyki) [inline src/main/groovy/com/sequenceiq/cloudbreak/client/CloudbreakClient.groovy]: Shouldn't the return variable be long? Besides that it looks good.

Comment (martonsereg) [inline src/main/groovy/com/sequenceiq/cloudbreak/client/CloudbreakClient.groovy]: I can change it but all the other post methods return String for some
strange reason :)

**Pull request merged**